variables.tf

```diff
@@ -63,3 +63,9 @@ variable "label_order" {
   default     = []
   description = "The naming order of the id output and Name tag"
 }
+
+variable "regex_allowed" {
+  type        = "string"
+  default     = "^[a-zA-Z0-9!@#$&()\\-`.+,/\"]*$"
```

Comment:
This should probably be `[^a-zA-Z0-9]` to preserve its original functionality

Reply:
@stephencoe is correct; we'll want to by default only allow alphanumeric characters; this is both for backwards compatibility, but also since 99% of cloud resources do not allow special characters in their names